ganttproject/src/net/sourceforge/ganttproject/search/SearchResultCellRenderer.java

```diff
@@ -0,0 +1,47 @@
+package net.sourceforge.ganttproject.search;

```

Comment:
Please add a copyright notice:
```
/*
Copyright 2018 BarD Software s.r.o, Douglas Kelly

This file is part of GanttProject, an opensource project management tool.
GanttProject is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.
GanttProject is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
*/
```